compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1500,44 +1500,37 @@ static LogicalResult setRootConfig(mlir::FunctionOpInterface entryPointFn,
 }
 
 /// Sets the lowering configuration for dispatch region for
-/// linalg_ext.winograd.input_transform root op.
+/// the three linalg_ext.winograd ops:
```

Comment:
nit: three linalg_ext.winograd ops: filter_transform, input_transform and output_transform.